tasks/CarryMyLuggage.tex

```diff
@@ -48,10 +49,10 @@ \subsection*{Additional rules and remarks}
 		\item a small object on the ground,
 		\item a hard-to-see 3D object,
 		and
-		\item a small area blocked using retractable barriers.
+		\item a small area blocked using retractable barriers. Teams can choose to have no obstacles placed if they do not want to go for bonus points. Teams can not pick and choose which obstacles are placed. Either none or all of them.
```

Comment:
"Retractable barrier" has [a specific meaning](https://www.amazon.com/Retractable-Wall-Barrier-Black-Finished/dp/B009YEE22U) that may or may not be what you were going for.

> Teams can choose to have no obstacles placed if they do not want to go for bonus points. Teams can not pick and choose which obstacles are placed. Either none or all of them.

The last two sentences are obvious enough from the first sentence that I think it's fine to drop them

Reply:
I did not add that but yes, that is what is meant there. Simply because we often have those at a competition.

Reply:
I would put it back:

"Teams can choose to have no obstacles placed if they do not want to go for bonus points. Teams can not pick and choose which obstacles are placed. Either none or all of them."

Because some teams expected these obstacles to be there but they weren't, at least as a reminder for TC to discuss during the Team Leaders meeting.